utils/generic.py

```diff
@@ -128,7 +128,7 @@ def get_frames(self) -> tuple:
             if running_time <= 1 / FRAMERATE:
                 sleepy_time = int(np.ceil(1000/FRAMERATE - running_time / 1000))
                 cv2.waitKey(sleepy_time)
-        else:
+        elif REPEAT_VIDEO:
```

Comment:
We need to do something with case when video ends to not end it on crash. Maybe have some exception here and catch it in dlc.py?

Reply:
Not sure how to do this. Can you make a suggestion?